packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2402,7 +2405,8 @@ class _InputDecoratorState extends State<InputDecorator> with TickerProviderStat
 
     final EdgeInsets contentPadding;
     final double floatingLabelHeight;
-    if (decoration.isCollapsed) {
+    if (decoration.isCollapsed
+        ?? themeData.inputDecorationTheme.isCollapsed) {
```

Comment:
NIT: can this fit a single LOC? I think it reaches 80 char exactly. But this is also fine.

Reply:
What would be a better way to refactor this?

Reply:
No, just personal

Reply:
I'm just asking as I'm open to suggestions for better code, nothing personal

Reply:
The split line seemed like dart format, but since you were already using a single line elsewhere, I thought it made more sense.. Thanks